app/src/main/java/de/triology/cas/ldap/CesAuthenticationEventExecutionPlanConfiguration.java

```diff
@@ -0,0 +1,226 @@
+package de.triology.cas.ldap;
```

Comment:
I thought we wanted to get rid of that package name in favour of something Cloudogu-specific?

Reply:
no, not right now.